src/witan/send/driver/ingest.clj

```diff
@@ -34,15 +34,17 @@
 (defn mover? [{:keys [setting-1 need-1 setting-2 need-2] :as rec}]
   (when (and (not (or (joiner? rec)
                       (leaver? rec)))
-             (or (not= setting-1 setting-2)
-                 (not= need-1 need-2)))
+             (not= setting-1 setting-2)
+             #_(or (not= setting-1 setting-2)
+                   (not= need-1 need-2)))
     rec))
 
 (defn stayer? [{:keys [setting-1 need-1 setting-2 need-2] :as rec}]
   (when (and (not (or (joiner? rec)
                       (leaver? rec)))
-             (and (= setting-1 setting-2)
-                  (= need-1 need-2)))
+             (= setting-1 setting-2)
+             #_(and (= setting-1 setting-2)
```

Comment:
I wonder if we should make this comply with a DSDR backed definition for mover and stayer and if it should be in witan.send rather than in witan.send.driver.

Reply:
The DSDR or the functions should be in witan.send?

Reply:
@seb231 I think both the DSDR and the functions should be in witan.send as I feel witan.send is what *owns* the domain.

Reply:
yeah okay. I'll come back to this then, as that seems like a bit more work